packages/@aws-cdk/aws-stepfunctions-tasks/test/apigateway/integ.invoke.ts

```diff
@@ -0,0 +1,48 @@
+import * as apigateway from '@aws-cdk/aws-apigateway';
+import * as lambda from '@aws-cdk/aws-lambda';
+import * as sfn from '@aws-cdk/aws-stepfunctions';
+import * as cdk from '@aws-cdk/core';
+import { ApiGatewayInvoke, AuthType, HttpMethod } from '../../lib';
+
+/*
+ * Stack verification steps:
+ * * aws stepfunctions start-execution --state-machine-arn <deployed state machine arn> : should return execution arn
+ * * aws stepfunctions describe-execution --execution-arn <exection-arn generated before> : should return status as SUCCEEDED and a query-execution-id
+ * * aws apigateway test-invoke-method --rest-api-id <value> --resource-id <value> --http-method <value>: should return the same response as the state machine
+ */
+const app = new cdk.App();
+const stack = new cdk.Stack(app, 'aws-stepfunctions-tasks-apigateway-invoke-integ');
+const restApi = new apigateway.RestApi(stack, 'MyRestApi');
+
+const hello = new apigateway.LambdaIntegration(new lambda.Function(stack, 'Hello', {
+  runtime: lambda.Runtime.NODEJS_10_X,
+  handler: 'index.handler',
+  code: lambda.Code.inline(`exports.handler = ${helloCode}`),
+}));
+
+function helloCode(_event: any, _context: any, callback: any) {
+  return callback(undefined, {
+    statusCode: 200,
+    body: 'hello, world!',
+  });
+}
```

Comment:
does this need to be a function? - why not just define it ahead of it's usage or inline it?